pages/linux/yum.md

```diff
@@ -2,10 +2,6 @@
 
 > Package management utility for RHEL, Fedora, and CentOS (for older versions).
```

Comment:
Could you add a link to a web man page? I found this:
```suggestion
> Package management utility for RHEL, Fedora, and CentOS (for older versions).
> More information: <https://man7.org/linux/man-pages/man8/yum.8.html>.
```
Do you know a better one?

Reply:
I looked for similar pages like [`apt`](https://github.com/tldr-pages/tldr/blob/cd3a80bffbd3f6c26c39e9513b5e072f58c9ddfa/pages/linux/apt.md) and [`apk`](https://github.com/tldr-pages/tldr/blob/5ef55e6fc44ddb75672469a9403e49702bc27753/pages/linux/apk.md), and they do not have a web man page link. I think the one you found is decent.